webapp/src/gitjson.tsx

```diff
@@ -801,6 +807,11 @@ class GithubComponent extends data.Component<GithubProps, GithubState> {
                     {!isBlocksMode && <ExtensionZone parent={this} needsToken={needsToken} githubId={githubId} master={master} gs={gs} isBlocks={isBlocksMode} needsCommit={needsCommit} user={user} pullStatus={pullStatus} pullRequest={pr} />}
                     <div></div>
                 </div>
+                <div className="ui serialHeader">
+                    <div className="rightHeader">
+                        {user && <sui.Button className="ui button" icon="fas fa-sign-out-alt" text={lf("Disconnect GitHub")} title={lf("Log out of GitHub")} onClick={this.handleSignoutGithub} onKeyDown={fireClickOnEnter} />}
```

Comment:
Are we using the `sui.Button` because it's already used within this file? I thought we preferred to use the react common components because they have better defaults for accessibility.

Reply:
Yeah, I wouldn't want to mix one react-common button in with 51 other sui elements on the page, nor swap over everything to react common just to add a single button on this page. Whenever we do a larger refactor on this page it should be done, though